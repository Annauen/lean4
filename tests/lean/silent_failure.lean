-- This was previously failing silently (but not adding `g` to the environment).

#guard_msgs (drop info) in
partial def g :
  have : False := by apply?
  False := g
