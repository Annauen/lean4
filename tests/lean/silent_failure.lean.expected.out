silent_failure.lean:4:12-4:13: error: invalid use of 'partial', 'g' is not a function
